deploy-board/deploy_board/templates/configs/new_capacity.html

```diff
@@ -21,7 +21,7 @@
         padding: 0 0 0 16px;
     }
 </style>
-<script type="text/javascript" src="{% static "js/components/sharedcomponents.js"%}?changedate=2016.12.19.150000"></script>
+<script type="text/javascript" src="{% static "js/components/sharedcomponents.js"%}?changedate=2023.01.30"></script>
```

Comment:
what is this changedate?

Reply:
It's a trick to force reload js. Browser can read from cache if the URL is the same.